Improve List demo description - itemTemplate (#564)
DevExpress/devextreme-demos: JSDemos/Demos/List/ItemTemplate/description.md

> The **List** supports item customization. You can use predefined customization settings. To do so, define [specific fields](/Documentation/ApiReference/UI_Components/dxList/Configuration/items/) in item data object.

Reviewer: I think we should elaborate on default templates (what you call "predefined customization settings") a bit more.

```suggestion
To customize list items, you can use a default item template or implement a custom template.

A default template is a set of data source fields that the List automatically recognizes. These fields are listed in the [items](/Documentation/ApiReference/UI_Components/dxList/Configuration/items/) section. To use the default template for customization, simply add or remove the fields to or from data objects. Refer to the [Default Templates](/Documentation/Guide/UI_Components/Common/Templates/#Default_Templates) article for more information.
```

> In cases when you need deeper customization, use the [itemTemplate](/Documentation/ApiReference/UI_Components/dxList/Configuration/#itemTemplate) property illustrated in this demo.

Reviewer: Since in my version, custom templates are mentioned as a second way of customization:

```suggestion
In cases when you need deeper customization, implement a custom template and assign it to the [itemTemplate](/Documentation/ApiReference/UI_Components/dxList/Configuration/#itemTemplate) property, as illustrated in this demo.
```

> To customize list items, you can use a default item template or implement a custom template.

Reviewer 2: Is this the first sentence of this topic? If so, to start straightaway with 'To do this' is not appropriate. A topic should have a suitable introduction. In any case, the first sentence of a paragraph or section should not start this way either.

 ```suggestion
You can use a default or custom item template to customize list items.
```

> A default template is a set of data source fields that the List automatically recognizes. These fields are listed in the [items](/Documentation/ApiReference/UI_Components/dxList/Configuration/items/) section. To use the default template for customization, simply add or remove the fields to or from data objects. Refer to the [Default Templates](/Documentation/Guide/UI_Components/Common/Templates/#Default_Templates) article for more information.

Reviewer 2: - "simply" and other adverbs should be avoided
- "add or remove the fields to or from" You do not need to add 'to or from' - **from** is sufficient here since it goes with **remove**

```suggestion
A default template consists of a set of data source fields that the List automatically recognizes. These fields are listed in the [items](/Documentation/ApiReference/UI_Components/dxList/Configuration/items/) section. To use a default template for customization, add or remove the fields from data objects. Refer to the [Default Templates](/Documentation/Guide/UI_Components/Common/Templates/#Default_Templates) article for more information.
```

> In cases when you need deeper customization, implement a custom template and assign it to the [itemTemplate](/Documentation/ApiReference/UI_Components/dxList/Configuration/#itemTemplate) property, as illustrated in this demo.

Reviewer 2: ```suggestion
If you want more customization options, implement a custom template and assign it to the [itemTemplate](/Documentation/ApiReference/UI_Components/dxList/Configuration/#itemTemplate) property, as illustrated in this demo.
```
